cmd/buildkitd/main_oci_worker.go

```diff
@@ -53,6 +54,21 @@ func init() {
 			Name:  "oci-worker-platform",
 			Usage: "override supported platforms for worker",
 		},
+		cli.StringFlag{
+			Name:  "oci-worker-net",
+			Usage: "worker network type (auto, cni or host)",
+			Value: defaultConf.Workers.OCI.NetworkConfig.Mode,
+		},
+		cli.StringFlag{
+			Name:  "oci-cni-config-path",
+			Usage: "path of cni config file",
+			Value: defaultConf.Workers.OCI.NetworkConfig.CNIConfigPath,
+		},
+		cli.StringFlag{
```

Comment:
why are you making two different paths for OCI and containerd cni binaries?  I think it would be safe to just have one dir

Reply:
from a data structure perspective workers should be isolated, eg. there should be a remote worker in the future that could run on a different machine (and different net config). So I want these values to be tied to the worker, not to the builder instance.